integration_tests/poetry.lock

```diff
@@ -777,7 +777,7 @@ tomlkit = "^0.7.0"
 type = "git"
 url = "https://github.com/crypto-com/pystarport.git"
 reference = "main"
-resolved_reference = "286a446473e1f3e1e1a5e075ba1041d3b52e5fc9"
+resolved_reference = "97e403875f31dd196e4062400a03358e66a13464"
```

Comment:
pystarport part seems correct, just cmd get changed for test gravity

Reply:
but the flags should be as the same as before

Reply:
got it, there's `yaml`s in `scripts` directory need to change as well, we probably should move them into `integration_tests/configs` directory ;D

Reply:
can, but those scripts are for non test usage right?

Reply:
[cronos-devnet.yaml](https://github.com/crypto-org-chain/cronos/blob/main/nix/default.nix#L29) still need, but we could update [test](https://github.com/yihuang/cronos/pull/6/files) only

Reply:
> can, but those scripts are for non test usage right?

yeah, only need one or two for example usage.